dc-geospatial-python/schedule.md

```diff
@@ -0,0 +1,35 @@
+<div class="row">
```

Comment:
I would suggest to put the online schedule here. This is the schedule that ends up in the workshop website. And next one will be online. See point 4 in https://github.com/esciencecenter-digital-skills/workshop-template#required-information-and-customizing-your-workshop-page for more information on different schedules (but you can also just do as I say ;) )

Reply:
Thanks @svenvanderburg ! I am using the online schedule now